workshops/4x4/index.html

```diff
@@ -0,0 +1,18 @@
+---
+layout: page
+title: 4x4 Keyboard Soldering Workshop
+subtitle: TBA • Living Computer Museum
+---
+<h2>Build your own 4x4 keyboard</h2>
+  <p>Learn the basics of keyboard electronics, microcontrollers, and soldering. Leave the workshop with your own 4x4 USB keyboard. Before you go, we will flash your keyboard with your choice as a num pad, emoji keyboard (Linux and Windows only), or midi controller. After the workshop you can learn how to program and flash your fully programmable keyboard yourself (or attend one of our not-yet-announced workshops on keyboard firmware). We will provide all tools and supplies.</p>
```

Comment:
Yeah, that's what the midi support does. The stuff I have built already doesn't use the midi mode, it is just "music mode" which let's you play audio through the built in speaker. I don't have anything (that I know of) to use actual midi to test, so if I were to build one it would be totally blind and untested.